project/plugins.sbt

```diff
@@ -1,12 +1,7 @@
 addSbtPlugin("com.github.sbt" % "sbt-unidoc" % "0.5.0")
-addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.5.9")
-addSbtPlugin("com.github.sbt" % "sbt-pgp" % "2.1.2")
+addSbtPlugin("org.typelevel" % "sbt-typelevel" % "0.4.3")
```

Comment:
This is now up to `0.4.9` if we want to update it before merging.

Reply:
Yeah, let me update it.